### ISSUE TITLE
Restore BMP support

### ISSUE THREAD

**Issue by minexew** [minexew/Shrine#8]
TempleOS update 877accc6598b79c75dc63bb9d5bc5e2ca446aab6 removes BMP support, replacing it with MV files and TOSZ functions that convert those into MP4.

We want BMP at the very least for Screenfetch, so we should either keep it in, or publish spin it off as a library.

**Comment by TomJansen** [comment]
why do you want bmp for screenfetch? Isn't png suitable?

**Comment by minexew** [comment]
PNG is not as straightforward, even without compression. It is against God's command.

**Comment by minexew** [comment]
Yeah, https://github.com/minexew/TempleOS/commit/877accc6598b79c75dc63bb9d5bc5e2ca446aab6
